### ISSUE TITLE
Is Wasserstein-distance calculated correctly?

### ISSUE THREAD

**Issue by LidaGuo1999** [jalola/improved-wgan-pytorch#28]
Hi, jalola

Thanks for your sharing. However, I find that your wasserstein-distance is calculated as d_fake-d_real. Shouldn't it be d_real-d_fake?

**Comment by jalola** [comment]
Hi @LidaGuo1999 ,

Right, probably it was copied from the loss line `disc_cost = disc_fake - disc_real + gradient_penalty` or we wanted to visualise it as a positive number. It does not affect the training.

**Comment by LidaGuo1999** [comment]
@jalola Yeah, this does not affect the training. Thanks.
